### PR TITLE
feat: explorer app cleanup

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Cleans up some unused files/hooks/functions in the explorer app
- Refactors `Media` usage to use the common lib version

## To test
- Making sure the responsive layout works correctly

### CODE REVIEW

**Review by alfetopito — COMMENTED** [review]
Looks good to me.
Just check the lint and build.

**Comment by elena-zh** [comment]
Hey @fairlighteth , could you please fix Explorer build so I can test it?

**Review by elena-zh — COMMENTED** [review]
Hey @fairlighteth , thank you!
However, I'm facing some issues: 
1. User details table: surplus field does not look good on [960-1300]px screen width 
image

2. Same with TX details page
image

All the rest changes LGTM!

**Comment by fairlighteth** [comment]
@elena-zh thanks for checking, can you check again?

**Comment by elena-zh** [comment]
@fairlighteth , looks the way better. However, I've noticed that a surplus value now is always displayed in the table. On prod right now we show it on a mobile resolution only: 
PR: 
image
Prod: 
image
 As far as I remember this was implemented this way in order not to overload the table with numbers on a limited column's width.

**Comment by fairlighteth** [comment]
I personally think it's working fine in the UI and simultaneously simplifies the code.

**Comment by elena-zh** [comment]
@fairlighteth , even in this case? 
image
or like [here](https://explorer-dev-git-explorer-cleanup-cowswap.vercel.app/gc/address/0xb93bdd291874d637abed0d7b86e31719031b5533)
image

compare with Prod: 
image

For me the table on prod looks cleaner a bit

**Comment by fairlighteth** [comment]
@elena-zh Could you share me the addresses with those orders? I can make sure they render with proper breaking. I do believe showing these values add value, given absolute amounts could say more than merely showing a percentage.

**Comment by fairlighteth** [comment]
@elena-zh I did a bunch of updates to the layout to improve things:

Screenshot 2024-07-02 at 10 42 22

https://github.com/cowprotocol/cowswap/assets/31534717/73ad985a-8afa-4aae-a54e-cea563b761f1

The only current bug that I can't seem to solve for now, is that on mobile/responsive there's this body horizontal scroll happening on the whole page:

https://github.com/cowprotocol/cowswap/assets/31534717/3a6e42a0-6cd7-468a-9043-73f40de92620

I don't think it's really blocking. I would suggest to make an issue for this and I will spend more time after to try to solve for this.

**Comment by elena-zh** [comment]
Hey @fairlighteth , great jib, thank you!

The issue I'd like to report you is that I cannot scroll the table to the left in a desktop view, and select 30-50 lines per page unless I scroll to the bottom and use the bar there. 
image

I don't like how an order details page looks like on a mobile resolution 
image


Then, on[ Fills tab ](https://explorer-dev-git-explorer-cleanup-cowswap.vercel.app/gc/orders/0x72198277f82a96d9ec2911c11f179d8605dedf5113a5194e98ac5235cdf6db709fa3c00a92ec5f96b1ad2527ab41b3932efeda58660ecabd/?tab=fills) cards do not cover the whole screen width, and the table uses the previous (cards) view for tablet/mobile resolutions
image

Contracts info is displayed like is not center-alignedimage
On Prod these links are left-aligned
image

Thanks

File: :q
Comment (alfetopito) [inline :q:1]: Here comes the vim exit again

Comment (fairlighteth) [reply :q:1]: I thought I deleted in my last commit :)

**Comment by alfetopito** [comment]
Linter again. Can you run `yarn lint --fix`? That should be enough 🤞

**Comment by fairlighteth** [comment]
@elena-zh Did a bunch of updates, please check again. One thing I want to address later is the mobile view of the order details. It works right now, using the horizontal scroll. I'd like to iterate on this in another PR to not have this horizontal scroll.

Screenshot 2024-07-04 at 14 44 53

**Review by alfetopito — APPROVED** [review]

**Pull request merged**